File: bus_station/src/main/java/com/thesis/bus_station/dto/BusSeatDTO.java
package com.thesis.bus_station.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class BusSeatDTO {

    public Long bus_id;
    public int seat_num;
    public String username;
    public boolean return_tic;
    public String type_of_discount;
    public double price;
    public String ticket_state;
}

File: Bus-station-react-frontend/src/dtos/BusSeatDTO.ts
export class BusSeatDTO {
    bus_id: number;
    seat_num: number;
    username:String;
    return_tic:boolean;
    type_of_discount:String;
    price:number;
    ticket_state:String;

    constructor(){
        this.bus_id = 0,
        this.seat_num = 0,
        this.username = "",
        this.return_tic = false,
        this.type_of_discount = "",
        this.price = 0,
        this.ticket_state = ""

    }
}